packages/components/src/external-link/index.tsx

```diff
@@ -65,18 +63,16 @@ function UnforwardedExternalLink(
 			target="_blank"
 			rel={ optimizedRel }
 			ref={ ref }
+			style={ { textDecoration: 'none' } }
 		>
-			{ children }
+			<span style={ { textDecoration: 'underline' } }>{ children }</span>
 			<VisuallyHidden as="span">
 				{
 					/* translators: accessibility text */
 					__( '(opens in a new tab)' )
 				}
 			</VisuallyHidden>
-			<StyledIcon
-				icon={ external }
-				className="components-external-link__icon"
```

Comment:
What happens with plugins that were relying on the `components-external-link__icon` classname to add styles to the icon?